README.md

```diff
@@ -10,60 +10,41 @@ The following sections detail how to make various improvements to the site. Make
 npm install
 ```
 
-### Documenting
+### Developing
 
 Run:
 
 ```
-npm run document
+npm run dev
```

Comment:
Is there a watch feature still or has it been removed?

Reply:
As reading package.json, I assume watch is now just the default. Might be worth mentioning that watch mode is on and there was a pro tip that was removed about styles/assets:

```
If you are making changes to styles, or adding images or other assets, you will need to run a `force` build while `skipping` installs with:

./node_modules/.bin/bit-docs -sfd

This should take about 5 seconds.
```

Is this tip still relevant?

Reply:
Watch mode is the default; I did remove the weird half-build that was "document" before. There's now just watch build on `dev` and full rebuild from scratch on `build`. Both of these will include style/image updates afaik, so I didn't think this one was relevant anymore.

Not opposed to adding a `rebuild-assets` command if you think it still has value though. (I haven't contributed to the academy from scratch in a while.)

Reply:
If the rebuilt for assets isn't needed anymore, then I'd skip it completely, but if it is something that isn't included by just doing `npm run dev` and there's a significant amount of time needed to run `npm run build` to get assets going, I'd add it back in some form.

Either way, if it's `npm run build` or `npm run build-assets` that is required to have updated assets/styles reflect what's being served, I'd include documentation on how to go about it.

Reply:
I put `rebuild-assets` and documented it, along with rewording around `dev` and `build` to clarify what they are for and what they do.